pyproject.toml

```diff
@@ -32,7 +35,7 @@ license = "MIT"
 name = "sindri"
 readme = "README.md"
 repository = "https://github.com/Sindri-Labs/sindri-python"
-version = "0.0.6"
+version = "0.0.0"
```

Comment:
I'm not sure what this `version` is all about, nor why you changed the string.

Reply:
This is the default version for the pypi upload. I changed it here to match our other default version, which is set [here](https://github.com/Sindri-Labs/sindri-python/pull/39/files#diff-d6ea7e6fc2faefc60dbe92247620b13c74212f731b76588bb466e174d93d2ab6R35).

In previous PRs, @sangaline and I configured the above linked version variable to be accessible/overwritten in the build/publish to pypi CI, so changing the default version here is a leftover fix.

Reply:
> In previous PRs, @sangaline and I configured the above linked version variable to be accessible/overwritten in the build/publish to pypi CI

Ahh yes I vaguely recall this. Thanks for explaining.